ci-operator/config/openshift/cloud-provider-powervs/openshift-cloud-provider-powervs-main.yaml

```diff
@@ -0,0 +1,60 @@
+base_images:
+  base:
+    name: "4.11"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.17-openshift-4.11:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.17-openshift-4.11
+binary_build_commands: go build .
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+images:
+- dockerfile_path: openshift-hack/images/Dockerfile.openshift
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.17-openshift-4.11:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.11
+  to: powervs-cloud-controller-manager
+promotion:
+  name: "4.11"
+  namespace: ocp
+releases:
+  initial:
+    integration:
+      name: "4.11"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.11"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: vet
+  commands: go vet $(go list ./... | grep -v "/vendor/")
+  container:
+    from: src
+- as: test
+  commands: go test -v -race -covermode=atomic -coverprofile=cover.out $(go list ./...)
```

Comment:
If you want prow to pick up your code coverage, it needs to be have `coverage` in the name and also be put into the `ARTIFACTS_DIR`, at the moment I don't think the cover.out file will go anywhere

Reply:
@JoelSpeed considering the nature of this repo(will get replaced with the IBM Cloud in the future), do you think is this coverage running here is necessary(the patch we are maintains in this repo is trivial one) or gives any value add?

Reply:
oops, its part of UT test job.. of course we need it but need coverage with this or not is the question :)

Reply:
I'm happy for this to merge as is, I was just working on fixing the coverage on another repo yesterday so thought I'd make a comment. I'm happy to add LGTM